### ISSUE TITLE
Timeline Scroll To

### ISSUE THREAD

**Issue by woodRock** [woodRock/woodrock.github.io#49]
## Goal 

When users click "see more" on the map page, the timeline scrolls to the relevant element. 

## Tasklist 

- [x] Anchor tag system for timeline elements. 
- [x] Hyperlink to timeline elements.
- [ ] (Optional) animated/smooth scrolling. 

## Success Criteria

The map links to the timeline smoothly, allowing seamless back-and-forth connected navigation, with support from #48.

**Comment by woodRock** [comment]
The react equivalent of an anchor tag is a reference. I implemented the following code snippet from stack overflow for the timeline component. This inspiration for this example is from [Stack Overflow](https://stackoverflow.com/questions/43441856/how-to-scroll-to-an-element). 

```js
const VTimeline = ({ markers, id }) => {
    const myRef = createRef(null);
    useEffect(() => {
      myRef.current.scrollIntoView();
    }, []);
    
    ... 

    return (
        <>
            ...
            {id === name && <div ref={myRef} />}
            ...
        </>
    )

```

The `id` prop is passed to the `VTimeline` component. We pass the selected marker between the `Map.js` and `Timeline.js` page using the react-router hooks.

```js
import { useParams } from "react-router-dom";
import VTimeline from "../components/Timeline.js";
...
const Timeline = () => {
  const { id } = useParams();
  return <VTimeline markers={markers} id={id} />
  ...
```

When we link to the timeline page from the `Map.js` page, we pass the name to the timeline route as an optional parameter.

```js
<Link to={"/timeline/" + name}> See more</Link>
```

We can specify optional routes in the `App.js` using the ? syntax. 

```js
...
<Router>
  ...
  <Route component={Timeline} exact path={"/timeline/:id?"} />
</Router>
```

**Comment by woodRock** [comment]
This issue was implemented in the commit here 4416fd4c54df7f41aaa2f68a2dbc3598777ec089
